docs/source/nb_api_intro.ipynb

```diff
@@ -128,7 +142,45 @@
     "The energy of the two $^1 A_{g}$ states can still be retrieved with the `value` function:\n",
     "```python\n",
     "fci.value('active space energy')[state] -> [-1.1083377195359851, -0.2591786932627466]\n",
-    "```\n"
+    "```\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8e9a09b6-36c0-453a-8f92-9f3ae6c08103",
+   "metadata": {},
+   "source": [
+    "## Multiconfigurational SCF (MCSCF)\n",
+    "\n",
+    "To run an MCSCF computation in which we simultaneously optimize the orbitals and active space wave function, we use the MCSCF class.\n",
+    "This class requires as an input node an `ActiveSpaceSolver` object, and therefore, can be combine with any of the solvers implemented in Forte.\n",
+    "The following shows a scheme of a typical CASCI computation, using Hartree–Fock orbitals as initial guess for the MCSCF computation\n",
+    "<center>\n",
+    "<img src=\"../source/img/api_intro-03.png\" alt=\"computation graph\" style=\"width: 636px;\"/>\n",
+    "</center>\n",
+    "\n",
```

Comment:
Please add a comment to the effect of "The graph does _not_ mean that there is a single ActiveSpaceSolver iteration, and then the ActiveSpaceSolver is never touched again. It means that the MCSCF object requires the ActiveSpaceSolver as an input (to solve for active space wavefunctions using the current orbitals), and the ActiveSpaceSolver requires the HF object as an input (to determine orbital space dimensions)."